mock_arduino.py

```diff
@@ -1,2 +1,10 @@
-def mock_arduino_response():
-    print("Your command did something!")
+from typing import Text
```

Comment:
I'd delete this import you don't need, then delete the empty lines below

Reply:
Done :)